src/styles/Nav.scss

```diff
@@ -0,0 +1,127 @@
+.selected{
+    color:#0275B1 !important;
+  }
+.icons>a, .user>a,.log-out>a{
+  text-decoration: none;
+  color: #181818;
+  &:hover, &:focus, &:active {
+    color:#0275B1;
```

Comment:
Please add a space between css attribute and value to keep the format consistent😆

Reply:
done